components/authz-service/server/v2/project.go

```diff
@@ -185,6 +185,7 @@ func (s *ProjectState) ApplyRulesStart(
 			"failed to parse state of rule apply")
 	}
 
+	s.log.Info("apply project rules: START")
```

Comment:
Drive-by: add back important log line that was inadvertently removed.

Reply:
💭 Important is in the eye of the beholder, I suppose. I'm not a fan of these context-less "this is happening" log messages. I'd prefer _debug_, I guess.

Reply:
My perspective: "we've got a problem with ingest rules--maybe it's because it was in the middle of the rule update? hmmm, can we see clearly in the log when that started and when that ended?" For a 12-hour operation the customer should not have to then say "Oops, nothing in the log. Maybe if we increased log verbosity to debug and try again...?"

Reply:
:smiIe: Are you sure _"apply project rules: START"_ is going to help **much** in that regard?

If this is for debugging failures, we should perhaps include much more: like, _which, or how many, staged rules are applied?_

Reply:
Fair question: yes, I think it will help, at least the first thing: if the customer is seeing something odd and they want to know if it was because an update was in progress or not, seeing when it _started_ and when it _finished_ is the first thing they need to figure out. I agree, though, that one could always give more info...